### PR TITLE
Add command mc!capacity to read and set the participant limit of preparing tournaments

### PR DESCRIPTION
**Pull request by kevinlul**
## Description

The generated query is
```sql
UPDATE "challonge_tournament" SET "participantLimit" = $2 WHERE "tournamentId" = $1;
```

## Checklist

- [x] I am following the [contributing guidelines](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/.github/CONTRIBUTING.md).
- [x] I have updated any relevant [user guides](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/guides).
- [x] I have updated any relevant [documentation](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/docs).
- [x] I have updated the [changelog](https://github.com/AlphaKretin/emcee-tournament-bot/blob/master/changelog.md), or this change is not user-facing.

### CODE REVIEW

File: src/commands/capacity.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			JSON.stringify({
+				channel: msg.channel.id,
+				message: msg.id,
+				user: msg.author.id,
+				tournament: id,
+				command: "capacity",
+				capacity
+			})
+		);
+		if (rawCapacity && (isNaN(capacity) || capacity < 0 || `${capacity}` !== rawCapacity)) {
```
Comment (AlphaKretin) [inline src/commands/capacity.ts:26]: This is a cool check on parseInt's results that I hadn't considered before

**Review by AlphaKretin — APPROVED** [review]
Is the plan to later transfer the TypeORM stuff to the database interface? Also maybe a comment explaining the default cap of 256 being challonge's end. Looks good in general.

File: src/commands/capacity.ts (hunk 1 above)
Comment (kevinlul) [reply src/commands/capacity.ts:26]: I used something like it in the YDK parser only to have to revert it when certain inputs had leading zeros

**Comment by kevinlul** [comment]
Not really unless it gets harder to test the code, since we have a lot of that boilerplate as-is. I'll find somewhere to document that it's an artifact of the free tier.

**Comment by kevinlul** [comment]
I can't tell which line I'm missing.

**Review by AlphaKretin — APPROVED** [review]

**Pull request merged**